tests/kafkatest/tests/security_rolling_upgrade_test.py

```diff
@@ -55,7 +55,7 @@ def create_producer_and_consumer(self):
             self.test_context, self.num_consumers, self.kafka, self.topic,
             consumer_timeout_ms=60000, message_validator=is_int, new_consumer=True)
 
-        self.consumer.group_id = "unique-test-group-" + str(random.random())
+        self.consumer.group_id = "group"
```

Comment:
I think originally @benstopford needed this group to change, but I'm not sure if that's still true?